src/CreaTeBME/connect.py

```diff
@@ -22,10 +21,17 @@ def connect():
             sys.exit(1)
     sensors = []
     for device in paired_devices:
-        sensor = ImuSensor(MODE_WIRELESS, device[0])
-        sensors.append(sensor)
-    return sensors
+        try:
+            sensor = ImuSensor(MODE_WIRELESS, device[0])
+            sensors.append(sensor)
+        except:
```

Comment:
Please only catch a specific error, as there is a possibility other errors still need to pass and/or exit the application.

Reply:
This only raises a RunTimeError and you cannot catch that as it is a base exception and even if I could I believe it too still be too vague to be meaningful to catch either way.